### PR TITLE
Read all entity observations from SQL by join observation with triples table on typeOf predicate

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: test/triples.csv [hunk 1]
```diff
@@ -36,3 +36,6 @@ test_var_3,typeOf,StatisticalVariable,
 test_var_3,measuredProperty,test_var_3,
 test_var_3,statType,measuredValue,
 test_var_3,name,,"non-place entity variable"
+ein/1,typeOf,Company,
```
Comment (keyurva) [inline test/triples.csv:39]: How are these files used? Are there tests that use them?

**Review by keyurva — APPROVED** [review]

File: internal/server/v2/observation/contained_in.go [hunk 2]
```diff
@@ -232,26 +223,58 @@ func FetchContainedIn(
...
+					AND o.variable IN (%s)
+				`,
+				childType,
+				variablesStr,
+			)
+			if queryDate != "" && queryDate != LATEST {
+				query += fmt.Sprintf("AND date = (%s) ", queryDate)
+			}
+			query += "ORDER BY date ASC;"
+			rows, err := store.SQLClient.Query(query)
```
[the remark this answers is not in the document]
Comment (shifucun) [reply internal/server/v2/observation/contained_in.go:247]: That's a really good point. Let me do it in a follow up PR.

File: test/triples.csv (hunk 1 above)
Comment (shifucun) [reply test/triples.csv:39]: Yes, these csvs are loaded into datacommons.db for testing via "./import.sh". I will document them too.

**Comment by shifucun** [comment]
thanks for review!

**Pull request merged**